src/components/Authentication/LoginPage.js

```diff
@@ -92,75 +122,317 @@ const LoginPage = () => {
+                                InputProps={{
+                                    startAdornment: (
+                                        <InputAdornment position="start">
+                                            <EmailOutlinedIcon style={{ color: "grey" }} />
+                                        </InputAdornment>
+                                    ),
+                                }} />
+                            <Grid container item className="buttonContainer">
+                                <Grid item md={2} />
+                                <Grid item md={4}>
+                                <Button className="createAccountButton"
                             component={Link} to={{
-                                "pathname": "/newaccount",
+                                "pathname": "/new/parent",
                                 "state": {
                                     email,
                                     password,
                                 },
                             }} variant="outlined">
-                            New Account
+                            CREATE ACCOUNT
                         </Button>
+                                </Grid>
+                                <Grid item md={4}>
+                                    <Button className="signInButton" onClick={() => handleCheck()}>
+                                        SIGN IN
+                                </Button>
+                                </Grid>
+                                <Grid item md={2} />
+                            </Grid>
+                        </Grid>
                     </Grid>
-                </Grid>
-            </form>
-            {hasError && (
-                <Typography color="error" data-cy="errorMessage">
-                    Invalid credentials
+                </form>
+            </div>)
+    }
+
+    const renderOtherLogins = () => {
+        switch (userType) {
+            case "Parent":
+                return (
+                    <div>
+                        <Ellipse3 className="ellipse3" />
+                        <Ellipse4 className="ellipse4" />
+                        <Picture2 className="picture1" />
+                        <div className="logo2">
+                            <Typography className="title">
+                                omou
                 </Typography>
-            )}
-        </Paper>
+                        </div>
+                        <form className="Login" onSubmit={handleLogin}>
+                            <Grid container>
+                                <Grid item md={6}></Grid>
+                                <Grid item md={6}>
+                                    <Typography className="welcomeText">
+                                        Hello Summit Parent
+                            </Typography>
+                                    <TextField error={hasError || email === ""} fullWidth
```

Comment:
Whenever you address the earlier comment to include checking if the email is valid, then you'll want to change this to just `{hasError}` instead of `{hasError || email === ""}`

Reply:
Make your email TextField its own component - similar to the PasswordInput component.